src/Microsoft.Win32.Registry/src/Microsoft/Win32/Registry.cs

```diff
@@ -14,7 +14,6 @@ namespace Microsoft.Win32
      * @security(checkClassLinking=on)
      */
     //This class contains only static members and does not need to be serializable.
-    [ComVisible(true)]
```

Comment:
One thing we will need to address as well is removing these from our contract reference assemblies. @davkean has been wanting to do that as well so perhaps he can help tackle that problem.